app/services/statement_classifier.rb

```diff
@@ -88,6 +88,8 @@ def statement_type(statement)
       :reconcilable
     elsif statement.unverifiable?
       :unverifiable
+    elsif (!statement.from_suggestions_store?) && statement.matches_but_not_checked?
```

Comment:
I could be wrong, but I don't think the braces are needed around the first condition here?

Reply:
No, I tend to over parenthesize to avoid someone reading the code having to think about precedence rules, but this one is clear enough without, you're right...